lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,24 @@
 import React, { Component } from 'react';
```

Comment:
Always check the console for warnings. Since Cars.js is a functional component, you can omit `{ Component }`.
```suggestion
import React from 'react';
```
will do just fine